union_pooling/union_pooling/union_pooler.py

```diff
@@ -77,6 +80,10 @@ def __init__(self,
     @param predictedActiveOverlapWeight: A multiplicative weight applied to
     the overlap between connected synapses and predicted-active-cell input
 
+    @param poolingActivationBurst: A fixed scalar amount of pooling activation
+    assigned to columns winning the inhibition step. If None, columns' pooling
```

Comment:
Indent wrapping lines. I usually do two indents (four spaces) to differentiate between indentation for wrapped lines and nested contexts from if statements or loops but some people do one indent